js/Source/Slider.Modify/Slider.Modify.js

```diff
@@ -75,9 +76,22 @@ Slider.Modify = new Class({
     }, this);
   },
 
+  addMoveClass: function(){
+    if(this.options.moveClass && this.options.moveClassTargets){
+      this.options.moveClassTargets.addClass(this.options.moveClass)
+    }
+  },
```

Comment:
I don't think this really belongs here. Your class is nicely abstract. You already have an event model here. This belongs in the behavior in an `onMove` event handler.